mf-dev/config.json

```diff
@@ -11,31 +11,31 @@
    },
    "configs": {
       "common": {
-         "footer": "/common/footer-config.json"
+         "footer": "/common/common_footer-config.json"
       },
       "dme": {
-         "auth": "/dme/auth-config.json",
-         "eventHandler": "/dme/event-handler-config.json",
-         "sidebar": "/dme/sidebar-config.json",
-         "mashupsList": "/dme/mashups-list-config.json",
-         "editor": "/dme/editor-config.json",
-         "editorAppbar": "/dme/editor-appbar-config.json",
-         "shareForm": "/dme/share-form-config.json",
-         "datasourcesList": "/dme/datasources-list-config.json",
-         "datasourceDetails": "/dme/datasource-details-config.json",
-         "flowsList": "/dme/flows-list-config.json"
+         "auth": "/dme/dme_auth-config.json",
```

Comment:
Yes I know...and you are right, but I decided to mantain these prefixes due to previous experience with the MINIO storage.
Because if you download a file or exchange a file config with others it appened that was uploaded a config from one application to another...and if files have the same name it is impossible understand where to find error.
With these prefix should be impossible these kind of errors.

Reply:
ok then. But, please, replace the underscore with a dash